source/_posts/2021-09-01-release-20219.markdown

```diff
@@ -0,0 +1,2682 @@
+
+Enjoy another great release! Be sure to tune into [the 2021.9 Release Party on YouTube](https://www.youtube.com/watch?v=CxrB7DmkDJQ) later today (9:00 PM CET) to learn more about this release!
+
+- [Home Energy Management updates](#home-energy-management-updates)
+  - [View energy usage over a period of time](#view-energy-usage-over-a-period-of-time)
+  - [Keeping track of your gas usage](#keeping-track-of-your-gas-usage)
+  - [Support for home batteries](#support-for-home-batteries)
+  - [Added support for many more integrations](#added-support-for-many-more-integrations)
+- [Long term statistics unlocked for all sensors](#long-term-statistics-unlocked-for-all-sensors)
+- [USB Discovery](#usb-discovery)
+- [Z-Wave JS Updates](#z-wave-js-updates)
+- [New template entities: Number and Select](#new-template-entities-number-and-select)
+- [New template functions for areas](#new-template-functions-for-areas)
+- [Other noteworthy changes](#other-noteworthy-changes)
+- [New Integrations](#new-integrations)
+- [Integrations now available to set up from the UI](#integrations-now-available-to-set-up-from-the-ui)
+- [If you need help...](#if-you-need-help)
+- [Breaking Changes](#breaking-changes)
+- [All changes](#all-changes)
+
+## Home Energy Management updates
+
+Last release, we [introduced the new Home Energy management features][energy]
+into Home Assistant and we are happy to see you like it!
+
+We have seen quite a few people sharing their Energy dashboards on socials like
+[Facebook][facebook] and [Twitter][twitter], which is really awesome.
+Share a screenshot of your dashboard and your story too!
+
+In this release, the story continues! Here is what we have in store for you:
+
+[energy]: /blog/2021/08/04/home-energy-management/
+[facebook]: https://www.facebook.com/groups/HomeAssistant
+[twitter]: https://twitter.com/home_assistant
+
+### View energy usage over a period of time
+
+The first iteration of the Energy Dashboard showed a single day at the time.
+Great! But what if you want to see how you did this week? Or month? Heck, a
+year even?!
+
+Well, now you can! In the top right of the dashboard, you can now select the
+grouping period you want to see: Day, Week, Month or Year.
+
+<p class='img'>
+<img src='/images/blog/2021-09/energy-month.png' alt='Screenshot of the new Energy Dashboard period selection'>
+Screenshot of the new Energy Dashboard period selection.
+</p>
+
+The arrow buttons allow you to browse through those periods. For example, when
+selecting a period of a month to display, the arrow buttons will take
+you a whole month back/forward in the view.
+
+### Keeping track of your gas usage
+
+In some countries, homes are connected to gas. The gas is being used to heat
+water, cook and for heating the house. If you have a home that is connected to
+gas, we have some great news for you!
+
+Maybe you noticed it already in the previous screenshot, as of this release,
```

Comment:
```suggestion
Maybe you noticed it already in the previous screenshot? As of this release,
```

Reply:
Not sure about this one, I like the current one better I think 🤷‍♂️